app/modules/entities/components/layouts/other_editions.svelte

```diff
@@ -31,12 +30,7 @@
   </div>
 {:then}
 	{#if isNonEmptyArray(otherEditions)}
-	  <div class="other-editions">
-	    <div class="editions-list-title">
-	      <h5>
-	        {I18n('other editions')}
-	      </h5>
-	    </div>
+    <div class="other-edition">
```

Comment:
Those dives just stack vertically, not using the available horizontal space on large screens. Could those `div.other-editions` be regrouped in a flex-row-wrapped list?
```suggestion
   <ul class="works-other-editions">
      <li class="other-editions">
```
```css
.works-other-editions{
  @include display-flex(row, baseline, center, wrap);
}
```

Reply:
done in 4f9a137